docs/visualization/plotting/plotting-nd-data.ipynb

```diff
@@ -191,8 +191,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d.coords['dummy-pos'] = sc.geometry.position(*[d.coords[dim] for dim in ['x', 'y', 'z']])\n",
-    "sc.plot(d, projection='3d', positions='dummy-pos')"
+    "d2 = d['y', :10].copy()\n",
+    "d2.coords['dummy-pos'] = sc.geometry.position(*[d2.coords[dim] for dim in ['x', 'y', 'z']])\n",
+    "sc.plot(d2, projection='3d', positions='dummy-pos')"
```

Comment:
Strange that before these changes, the pixels looked like they had the correct size (1.0)
![Screenshot at 2021-12-02 08-02-09](https://user-images.githubusercontent.com/39047984/144374009-79c1297c-9f1f-4caa-8266-baf9230e7bf6.png)
but now they look too small?
![Screenshot at 2021-12-02 08-02-39](https://user-images.githubusercontent.com/39047984/144374059-74e1bf71-ee4b-4ee1-8d21-d45cec9739e6.png)

Reply:
Can't tell why. Maybe just coincidence?

Reply:
Scaling the positions has no influence, so it is probably related to the number of points getting plotted? In any case, probably unrelated to this particular change, i.e., a generic problem(?)?

Reply:
Yes, it's unrelated. See https://github.com/scipp/scipp/pull/2310